azure-pipelines.yml

```diff
@@ -29,14 +29,25 @@ jobs:
   - pwsh: ./tools/start-emulators.ps1 -NoWait
     displayName: "Start emulators (-NoWait)"
 
+  - pwsh: ./install-dotnet.ps1 -Version '6.0.100-rc.2.21505.57'
```

Comment:
Could you follow the pattern I'm using in the host for this by using a yaml template? It makes sure there's only one place we need to update this if we have to move versions.

https://github.com/Azure/azure-functions-host/blob/v4.x/azure-pipelines.yml#L45
https://github.com/Azure/azure-functions-host/blob/v4.x/build/install-dotnet.yml

Reply:
Thank you @brettsam for your review and feedback. 

Currently, https://github.com/Azure/azure-functions-host/blob/v4.x/build/install-dotnet.yml does not cover the Linux scenario, so we would have to add the equivalent of this script `install-dotnet.ps1`  in yml. 

Would you be ok if we set the version to be installed in the script itself. This way, we can call the script without parameters, e.g., 

`- pwsh: ./install-dotnet.ps1`


```
param(
    [String]
    $Version = '6.0'

    [String]
    $Channel = 'release'
)

$obtainUrl = "https://raw.githubusercontent.com/dotnet/cli/master/scripts/obtain"
try
{
    $installScript = if ($IsWindows) { "dotnet-install.ps1" } else { "dotnet-install.sh" }
    Invoke-WebRequest -Uri $obtainUrl/$installScript -OutFile $installScript

    if ($IsWindows)
    {
        & .\$installScript -Channel $Channel -Version $Version -InstallDir "$env:ProgramFiles/dotnet"
    }
    else
    {
        bash ./$installScript -c $Channel -v $Version --install-dir "/usr/share/dotnet"
    }
}
finally
{
    Remove-Item $installScript -Force -ErrorAction SilentlyContinue
}
```

Alternatively, if we decide to go the yml route, this is how the code would look like:

```
- pwsh: |
    Invoke-WebRequest 'https://raw.githubusercontent.com/dotnet/cli/master/scripts/obtain/dotnet-install.ps1' -OutFile 'dotnet-install.ps1'
    ./dotnet-install.ps1 -InstallDir "$env:ProgramFiles/dotnet" -Version "6.0" -Channel 'release'
  displayName: 'Install the .Net version used by the Core Tools for Windows'
  condition: eq( variables['Agent.OS'], 'Windows_NT' )

- bash: |
    curl -sSL https://raw.githubusercontent.com/dotnet/cli/master/scripts/obtain/dotnet-install.sh | bash /dev/stdin -v '6.0' -c 'release' --install-dir /usr/share/dotnet
  displayName: 'Install the .Net version used by the Core Tools for Linux'
  condition: eq( variables['Agent.OS'], 'Linux' )
```

Thoughts?

Reply:
I like the yaml route better -- (i'd like to move more things, like signing, out so it keeps the core yaml smaller)

Reply:
Sounds good. I will move the code to yml.